src/task 25/main.js

```diff
@@ -0,0 +1,10 @@
+var arr = ["81","87","88"];
```

Comment:
Far more better would be to path characters, not its codes, and them translate in the function code. Also you should handle case insensivity.